docs/source/conf.py

```diff
@@ -47,6 +47,7 @@
     'sphinx.ext.ifconfig',
     'sphinx.ext.githubpages',
     'sphinx.ext.autodoc',
+    'sphinx.ext.napoleon',
```

Comment:
**thought:** While this fixes the RTD build, it might be a better idea to provide our own docstrings...